test_all.bat

```diff
@@ -0,0 +1,19 @@
+ECHO running unit tests
```

Comment:
Is this a leftover?

Reply:
I mean the whole file

Reply:
i found myself pretty smart for coming up with this short cut... spares me doing this manually every time I want to check before pushing.  
Should I remove it?

Reply:
keep it if it's useful. You can put it in a folder.

Reply:
done.  
or should I add `.bat` to the gitignore?

Reply:
I leave this up to you. If you use only one computer, you don't have to put it under version control.

Reply:
I use two... so if it doesn't bother you, I'd leave it in. :)